src/components/AddStudent.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import Cookies from 'js-cookie';
+import { ToastContainer, toast } from 'react-toastify';
+import { Link } from "react-router-dom";
+import Button from '@mui/material/Button';
+
+// jshint ignore: start
+export default class AddStudent extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            name: "",
+            email: ""
+        };
+    }
+
+    handleClickOpen = () => {
+        this.setState({ open: true });
+    };
+
+    handleClose = () => {
+        this.setState({ open: false });
+    };
+
+    handleChange = (event) => {
+        console.log(this.state);
+        this.setState(() => {
+            return {
+                [event.target.name]: event.target.value
+            }
+        })
+    }
+
+    onSubmit = () => {
+        console.log("Submit attempt");
```

Comment:
Nice touch on the logs, to help the reader and developer know if a request is going through the intended path, or if an error occured